sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/AnomalyDetectionConfigurationLiveTests.cs

```diff
@@ -14,6 +14,10 @@ namespace Azure.AI.MetricsAdvisor.Tests
 {
     public class AnomalyDetectionConfigurationLiveTests : MetricsAdvisorLiveTestBase
     {
+        private const string MetricName = "metric";
+        private const string DimensionNameA = "DimensionA";
+        private const string DimensionNameB = "dimensionB";
```

Comment:
For our validations, we were using the static data feed we have in our resource. We assumed the metric "cost" and the dimension names "category" and "city". However, we're creating a mock data feed to contain our configurations now. We're also specifying these mock names when creating the data feed and validating the configurations.

Reply:
nit: caps on values?

Reply:
Updated! Not sure what happened to my brain here.